### ISSUE TITLE
Archetype generation uses system java, instead Eclipse's

### ISSUE THREAD

**Issue by fbricon** [eclipse-m2e/m2e-core#785]
I tried #249 with the faktor-ips archetype, looks like my system was set to use Java 7 (due to prior testing). Archetype generation failed:

<img width="1340" alt="Screenshot 2022-06-15 at 12 37 57" src="https://user-images.githubusercontent.com/148698/173808473-b3bcc594-997e-41a8-b7ff-9256f648751f.png">

IMO it should have used whatever Java version is currently running Eclipse

**Comment by laeubi** [comment]
Can you check `Preferences > JDK`  what is selected as the default JDK? I think that is what m2e uses.

**Comment by fbricon** [comment]
Right, Java 7 was set as the default JRE. So it's easy to fix once you know it, but I think it'd be safer to use whatever JDK is running Eclipse, since it'll be 17 minimum now.
Not a high priority bug though.

**Comment by laeubi** [comment]
@fbricon that is default behavior of the m2e launcher that the archetype uses internally, so would need to be a general change.

**Comment by fbricon** [comment]
I think it would be good to have that behavior documented somewhere
